scanner/src/lib/dataproducts/Battery_ManufacturingDataSheet_v0.1.svelte

```diff
@@ -0,0 +1,251 @@
+  <DataRow label="Minimum" value={`${formatNumber(data.voltageLevels.minimumVoltage, "V")}`} />
+  <div class="divider" />
+  <div class="title no-bottom-margin">Temperature range</div>
+  <div class="subtitle">The details of the acceptable temperature values of the battery</div>
+  <DataRow
+    label="Temperature range"
+    value={`${formatNumber(data.temperatureRange.minimumTemperature, "°C")} to ${formatNumber(
+      data.temperatureRange.maximumTemperature,
+      "°C"
+    )}`}
+  />
+  <div class="divider" />
+  <div class="title no-bottom-margin">Expected lifetime</div>
+  <div class="subtitle">The details of the battery lifetime</div>
+  <DataRow label="Cycle life" value={formatNumber(data.expectedLifetime.cycleLife)} />
+  <DataRow label="Reference test" value={data.expectedLifetime.referenceTest} />
+  <DataRow label="Cycle rate" value={data.expectedLifetime.cycleRate} />
+  <div class="divider" />
+  <div class="title no-bottom-margin">Material composition</div>
+  <div class="subtitle">The details of the material composition of the battery</div>
+  <DataRow label="Chemistry" value={data.materialComposition.chemistry.join(", ")} />
+  <DataRow
+    label="Hazardous substances"
+    value={data.materialComposition.hazardousSubstances.join(", ")}
+  />
+  <DataRow
+    label="Critical raw materials"
+    value={data.materialComposition.criticalRawMaterials.join(", ")}
+  />
+  <div class="divider" />
+  <div class="title no-bottom-margin">Recycled content</div>
+  <div class="subtitle">The recycled content information present in the battery</div>
+  {#if data.recycledContent.length > 0}
+    <ul>
+      {#each data.recycledContent as content}
+        {#if content.substanceName.trim() && content.recyclingRate}
+          <li class="list-item">
+            {content.substanceName}: {formatNumber(content.recyclingRate, "%")}
+          </li>
+        {/if}
+      {/each}
+    </ul>
+  {:else}
+    <p>-</p>
+  {/if}
+  <div class="divider" />
+  <div class="title no-bottom-margin">Renewable content</div>
+  <div class="subtitle">The renewable content information present in the battery</div>
+  {#if data.renewableContent.length > 0}
+    <ul>
+      {#each data.renewableContent as content}
+        {#if content.substanceName.trim() && content.proportion}
+          <li class="list-item">
+            {content.substanceName}: {formatNumber(content.proportion, "%")}
+          </li>
+        {/if}
+      {/each}
+    </ul>
+  {:else}
+    <p>-</p>
+  {/if}
+  <div class="divider" />
+  <div class="title no-bottom-margin">Legal conformity</div>
+  <div class="subtitle">
+    The details of the conformity of the battery with the legal and harmonized standards
+  </div>
+  <div class="act-compliance">
+    <p class="label">Battery act compliance:</p>
+    <div class="compliance-img">
+      <img src={data.legalConformity.batteryActCompliance ? True : False} alt="compliance status" />
```

Comment:
the `alt` here adds zero value, it should say `Complies with the battery act` or `Does not comply with the battery act` accordingly.